firefox-ios/firefox-ios-tests/Tests/ClientTests/Microsurvey/Mock/MicrosurveyPromptMiddlewareTests.swift

```diff
@@ -9,6 +9,8 @@ import XCTest
 
 final class MicrosurveyPromptMiddlewareTests: XCTestCase {
     private var microsurveyManager: MockMicrosurveySurfaceManager!
```

Comment:
I know this isn't related to your changes, but maybe we should rename this to `mockMicrosurveyManager` for clarity? 🤔  Otherwise it looks like we're passing in a real instance at first glance in the tests below.